### DISCUSSION TITLE
Welcome to the Big Bee Community Discussion Forum!

### DISCUSSION THREAD

**Discussion by seltmann** [Big-Bee-Network/Big-Bee-community-discussion#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Post opportunities!
  * Engage with other community members.
  * Welcome others and are open-minded and bee-minded. Remember that this is a community we
  build together 💪 .
  * This is an open community forum and the information posted here should be about bee-related activities and interests. The maintainers expect posts to be curious and inclusive.

**Comment by dryopoiddarling** [comment]
Hi All! This is Crystal Maier!

**Reply by seltmann** [reply]
Welcome to the discussion forum!
